pleskdistup/actions/common_checks.py

```diff
@@ -242,6 +242,54 @@ def _do_check(self) -> bool:
         return False
 
 
+class AssertOSVendorPHPUsedByWebsites(action.CheckAction):
```

Comment:
Please capitalize only the first letter of each word, like `AssertMinPhpVersionUsedByCron` above.